java/java.lsp.server/vscode/src/extension.ts

```diff
@@ -274,54 +282,82 @@ function activateWithJDK(specifiedJDK: string | null, context: ExtensionContext,
                 'nbcodeCapabilities' : {
                     'statusBarMessageSupport' : true
                 }
+            },
+            errorHandler: {
+                error : function(_error: Error, _message: Message, count: number): ErrorAction {
+                    return ErrorAction.Continue;
+                },
+                closed : function(): CloseAction {
+                    activateWithJDK(specifiedJDK, context, log, false);
```

Comment:
This should contribute a lot to the robustness of the NetBeans backend integration. Previously it was not guaranteed the launch process really goes thru the `activateWithJDK` booting sequence. By default (if there is no `errorHandler`) the NetBeans client is restarted automatically by trying to connect to dead NetBeans instance.

With this change a restart should replay the whole booting sequence in `activateWithJDK`.